.travis.yml

```diff
@@ -1,9 +1,22 @@
-language: vim
+language: ruby
+rvm:
+  - 1.8.7 # Use stock
+  - 1.9.2 # Build latest
 
-install: |
-  sudo apt-get update
-  sudo apt-get install vim
+before_script: |
   cd ..
+  if [ $(ruby -e 'puts RUBY_VERSION') == '1.8.7' ]; then
+    sudo apt-get -y update
+    sudo apt-get -y install vim
+    sudo ln -s /usr/bin/vim /usr/local/bin/vim
+  else
+    git clone --depth 1 https://github.com/vim/vim
```

Comment:
It would also be cool to checkout to the latest tag stable before building: http://stackoverflow.com/questions/17414104/git-checkout-latest-tag